packages/@unimodules/core/android/src/main/java/org/unimodules/core/ModuleRegistry.java

```diff
@@ -93,25 +91,24 @@ public InternalModule unregisterInternalModule(Class exportedInterface) {
 
   public void registerExportedModule(ExportedModule module) {
     String moduleName = module.getName();
-
     mExportedModulesMap.put(moduleName, module);
     mExportedModulesByClassMap.put(module.getClass(), module);
-    maybeAddRegistryConsumer(module);
   }
 
   public void registerViewManager(ViewManager manager) {
     String managerName = manager.getName();
-
     mViewManagersMap.put(managerName, manager);
-    maybeAddRegistryConsumer(manager);
   }
 
   public void registerSingletonModule(SingletonModule singleton) {
     String singletonName = singleton.getName();
-
     mSingletonModulesMap.put(singletonName, singleton);
   }
 
+  public void registerOuterListener(ComponentLifecycleListener outerListener) {
```

Comment:
I'm wondering if `registerExtraListener` or just `registerListener` wouldn't be better?